packages/grid/_modules_/grid/components/cell/GridCell.tsx

```diff
@@ -35,9 +35,10 @@ export interface GridCellProps {
   value?: GridCellValue;
   width: number;
   cellMode?: GridCellMode;
+  children: React.ReactNode;
```

Comment:
It's a small thing, but we could narrow it a little more. `GridCell` always receives a `ReactElement` or null from `GridRowsCell`.

```suggestion
  children: React.ReactElement | null;
```

Reply:
Sure. Added